recipe/test/test_iostreams_zlib.cpp

```diff
@@ -0,0 +1,19 @@
+// Ensure that boost::iostreams includes the zlib filter support
+
+#include <fstream>
+#include <iostream>
+#include <boost/iostreams/filtering_streambuf.hpp>
+#include <boost/iostreams/copy.hpp>
+#include <boost/iostreams/filter/zlib.hpp>
+
+int main()
+{
+    using namespace std;
+
+    ifstream file("hello.z", ios_base::in | ios_base::binary);
+    boost::iostreams::filtering_streambuf<boost::iostreams::input> in;
+    in.push(boost::iostreams::zlib_decompressor());
+    in.push(file);
+    boost::iostreams::copy(in, cout);
```

Comment:
Maybe it is worth printing some output just to know what is happening in the logs (and help diagnose any issues if they arise)?

Reply:
We only compile and link this program - we don't actually run it. So you wouldn't see any prints anyway.

Reply:
Ah ok. Should we try running it then?

Reply:
In principle that should work (and would give you some log output, since `hello.z` gets dumped to `stdout`) - I'll take a look.